<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>484</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="triangle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Трикутник</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>За формулою Герона</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Довжини двох сторін і значення кута між ними</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string>Довжина сторони і опущена на неї висота</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Довжини трьох сторін і радіус вписаного кола</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>Довжини трьох сторін і радіус описаного кола</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Довжина одного боку і двох кутів</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>501</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Оберіть варіант обчислення площі трикутника, відповідно до відомих данних</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>281</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>270</y>
        <width>801</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>290</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>201</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Чотирикутник</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Квадрат</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Прямокутник</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Паралелограм</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Ромб</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Трапеція</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Круг</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Еліпс</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
